### PR TITLE
Small translation fixes

### PR DESCRIPTION
**Pull request by Fridolin012**
## Type of PR

- [X] Translation update / creation
- [ ] Original file creation
- [ ] Original file content (not typo) update
- [X] Original file typo update

## Checks

If you have changed the content (/ meaning) of an original file, please confirm:

- [ ] Yes, I have copied the updated section to all translations of this file manually.

## Notes

Please enter additional information here.

### CODE REVIEW

File: i18n/de/docusaurus-plugin-content-docs/current/scnx/account-and-billing/faq.md [hunk 1]
```diff
@@ -63,10 +63,9 @@ Bitte lies dir diesen [FAQ-Artikel](https://faq.scnx.app/de/sepa-lastschrift/) d
 
 Wenn du per Banküberweisung bezahlst, wird dir eine individuelle IBAN auf einem virtuellen Bankkonto zugewiesen. Jede Zahlung, die auf diesem Konto eintrifft, wird deinen Rechnungen gutgeschrieben – die Angabe der Transaktionsreferenz ist nicht erforderlich (wird jedoch empfohlen bei Einmalkäufen empfohlen), da wir anhand der IBAN, auf der das Geld eingegangen ist, eine Zahlung problemlos deinem Konto zuordnen können.
 
-Durch einen Unfall zu viel Geld verschickt? Keine Sorge: Dein Geld wird automatisch für zukünftige Zahlungen verwendet.
-von (falls dies nicht möglich ist, wird sich Stripe mit dir in Verbindung setzen und nach den Bankdaten fragen). Du kannst
-von (falls dies nicht möglich ist, wird sich Stripe mit dir in Verbindung setzen und nach den Bankdaten fragen). Du kannst
- (falls dies nicht möglich ist, wird sich Stripe mit dir in Verbindung setzen und nach den Bankdaten fragen). Du kannst
+Durch einen Unfall zu viel Geld verschickt? Keine Sorge: Dein Geld wird automatisch für zukünftige Zahlungen verwendet (falls dies nicht möglich ist, wird sich Stripe mit dir in Verbindung setzen und nach den Bankdaten fragen). Andernfalls kannst
```
Comment (SCDerox) [inline i18n/de/docusaurus-plugin-content-docs/current/scnx/account-and-billing/faq.md:66]:
```suggestion
Aus Versehen zu viel Geld verschickt? Keine Sorge: Dein Geld wird automatisch für zukünftige Zahlungen verwendet (falls dies nicht möglich ist, wird sich Stripe mit dir in Verbindung setzen und nach den Bankdaten fragen). Andernfalls kannst
```

File: i18n/de/docusaurus-plugin-content-docs/current/scnx/guilds /custom-bot/modules/administration /logging.md [hunk 2]
```diff
@@ -1,71 +1,71 @@
...
 
-Lass deinen Bot alle Änderungen an deinem Sever und deinen Kanälen Protokollieren.
+Lass deinen Bot alle Änderungen an deinem Sever und deinen Kanälen protokollieren.
 
 <ModuleOverview moduleName="logging" />
 
 ## Funktionen {#features}
 
-Mit dem erweiterten Protokollierungsmodul kannst du dies ganz einfach tun
+Mit dem Fortgeschrittenes Logging-Modul kannst du folgendes ganz einfach tun:
```
Comment (SCDerox) [inline i18n/de/docusaurus-plugin-content-docs/current/scnx/guilds /custom-bot/modules/administration /logging.md:9]:
```suggestion
Mit dem fortgeschrittenen Logging-Modul kannst du folgendes ganz einfach tun:
```
Name of the module

**Review by SCDerox — CHANGES_REQUESTED** [review]
Two small things

**Review by SCDerox — APPROVED** [review]

**Comment by Fridolin012** [comment]
Hello there

**Pull request merged**